test/postman/inference_data.json

```diff
@@ -221,14 +221,28 @@
         "model_name":"fcn_resnet_101",
         "worker":1,
         "synchronous":"true",
-        "file":"../examples/object_detector/persons.jpg"
+        "file":"../examples/image_segmenter/persons.jpg"
     },
     {
         "url":"https://torchserve.pytorch.org/mar_files/fcn_resnet_101_scripted.mar",
         "model_name":"fcn_resnet_101_scripted",
         "worker":1,
         "synchronous":"true",
-        "file":"../examples/object_detector/persons.jpg"
+        "file":"../examples/image_segmenter/persons.jpg"
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/deeplabv3_resnet_101.mar",
```

Comment:
Should I also update this URLs? Let me know 😄

Reply:
yes, thanks.

Reply:
Done! 💪🏻